app/services/openlibrary_service.rb

```diff
@@ -0,0 +1,17 @@
+class OpenLibraryService
+  include HTTParty
+  base_uri 'openlibrary.org'
+  def initialize
+    @options = { query: { format: 'json', jscmd: 'data' } }
+  end
+
+  def book_info(isbn)
+    ibsn_formatted = 'ISBN:'.concat(isbn)
+    @options[:query][:bibkeys] = ibsn_formatted
```

Comment:
Create a method that returns the request body

Reply:
Done!